react-basic-practice/src/index.css

```diff
@@ -1,8 +1,6 @@
 @font-face {
   font-family: 'Inter';
-  src: url('../src/assets/fonts/Inter-Regular.ttf') format('woff2'),
-       url('../src/assets/fonts/Inter-Regular.ttf') format('woff'),
-       url('../src/assets/fonts/Inter-Regular.ttf') format('truetype');
+  src: url('./assets/fonts/Inter-Regular.ttf') format('truetype');
```

Comment:
Please refer to using define forn: https://v2.chakra-ui.com/community/recipes/using-fonts